docs/docs/changelogs/v15.1.0.md

```diff
@@ -0,0 +1,99 @@
+---
+title: v15.1.0
+order: 1
+---
+
+This document outlines API breaking changes for that were introduced in `gaia v15.1.0`.
+
+This release is based on cosmos-sdk `v0.47.x`.
+
+You can find the comprehensive API docs at:
+* https://docs.cosmos.network/api
+
+Module refrence manuals (with CLI instructions) can be found at:
+* https://docs.cosmos.network/v0.47/build/modules
+
+
+# Important changes
+Changes are outlined compared to `gaiad <= v14.x`
+
+## Behaviour changes
+
+Starting `v15.1.0` all users must have at least `1 ATOM` staked in order to cast a vote on a governance proposal.
+* votes from accounts whose staked amounts are `< 1 ATOM` will be rejected
```

Comment:
Why is this a bullet point? Is this for emphasizing the importance? Why not use `:::tip  :::`?